File: fe2-android/app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/election/ElectionEnd.java
package com.github.dedis.student20_pop.model.network.method.message.data.election;

import com.github.dedis.student20_pop.model.network.method.message.data.Data;
import com.google.gson.annotations.SerializedName;

public class ElectionEnd extends Data {

    @SerializedName(value = "election")
    private String electionId;
    @SerializedName(value = "created_at")
    private long createdAt;
    @SerializedName(value = "lao")
    private String laoId;


    @Override
    public String getObject() {
        return null;
    }

    @Override
    public String getAction() {
        return null;
    }
}

File: be1-go/message2/messagedata/election_end.go
package messagedata

// ElectionEnd ...
type ElectionEnd struct {
	Object          string `json:"object"`
	Action          string `json:"action"`
	LAO             string `json:"lao"`
	Election        string `json:"election"`
	CreatedAt       int    `json:"created_at"`
	RegisteredVotes string `json:"registered_votes"`
}
